enhancements/results-forwarding.md

```diff
@@ -0,0 +1,379 @@
+---
+title: results-forwarding
+authors:
+  - TBD
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - TBD
+approvers:
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Compliance Results Forwarding / Storage
+
+## Summary
+
+This will allow for a flexible and extendible mechanism to forward compliance
+results and evidence to centralized stores, respectively.
+
+## Motivation
+
+While the current approach of generating results and remediations via CRDs
+has worked so far for Compliance Operator, this has several draw-backs that
+prove painful in multi-cluster environments:
+
+* `ComplianceCheckResults` and `ComplianceRemediations` exist in the cluster
+  that's the target for compliance checks, when expanding operations to multiple
+  clusters, this becomes cumbersome to check.
```

Comment:
Cumbersome because each CO is only aware of itself and checks/results are not bound to the cluster, right?

That's what comes up to me when I think of this pain point, but we could elaborate on more of them here if there are multiple.